### ISSUE TITLE
Unable to use Mass Edit to change profile effectively

### ISSUE THREAD

**Issue by bifa2** [morpheus65535/bazarr#1636]
**Describe the bug**
I am trying to use Mass Edit to change the profile for my imported movies as sadly this wasn't done on import and it's preventing me searching for subtitles.

When clicking Mass Edit, the UI fails to display all movies - this may be by design, only showing the page you're on but 
1) I've 61 pages of results to edit the profile for 😔
2) Trying to edit any page > 1 just results in the UI reloading and mass edit showing the results from page 1.

**To Reproduce**
1. Go to http://localhost:6767/movies
2. Navigate to a page of results further into your listing.
3. Click on 'Mass Edit'
4. See the results from page 1 :(

**Expected behavior**
Ability to mass edit **all** movies at once, or at least the UI letting you edit the page you were on.

**Software (please complete the following information):**
  - Bazarr Version: 1.0.1
  - Radarr Version: 3.2.2.5080
  - Operating System: Linux-5.10.60.1-microsoft-standard-WSL2-x86_64-with
  - Python Version: 3.9.5

**Comment by bifa2** [comment]
Infact, it's even weirder. I navigated to page 58 and clicked Mass Edit and the result is what looks to be page 1 and 2 as it's into films starting with 'A' but then jumps to results starting from 'So' and continues.

**Comment by bifa2** [comment]
Another test. Navigate to page 6, click Mass Edit, get results from page 1 to 6 (ending with 'Belle'), and then jumps once more to 'So' onwards .

**Comment by morpheus65535** [comment]
When you click Mass edit, you get the full list of movies. I've just tested and it works.

**Comment by bifa2** [comment]
Not for me. What I can I provide (the logs don't seem to show anything).

**Comment by bifa2** [comment]
My mistake, they do. It seems like an error 500 is terminating the return:

![image](https://user-images.githubusercontent.com/54040596/145472601-7b0d084f-d197-4522-bba2-8bcf91755418.png)

```
Exception on /api/movies [GET] | 09/12/2021 20:42:06
```

```
Traceback (most recent call last):

  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1949, in full_dispatch_request

    rv = self.dispatch_request()

  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1935, in dispatch_request

    return self.view_functions[rule.endpoint](**req.view_args)

  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 458, in wrapper

    resp = resource(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 89, in view

    return self.dispatch_request(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 573, in dispatch_request

    resp = meth(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/api.py", line 78, in wrapper

    return actual_method(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/api.py", line 955, in get

    postprocessMovie(item)

  File "/app/bazarr/bin/bazarr/api.py", line 250, in postprocessMovie

    item[\'subtitles\'] = sorted(item[\'subtitles\'], key=itemgetter(\'name\', \'forced\'))

TypeError: \'<\' not supported between instances of \'str\' and \'NoneType\
```

**Comment by bifa2** [comment]
Found someone with what appears to be the same issue: https://forums.unraid.net/topic/74410-support-linuxserverio-bazarr/?do=findComment&comment=1001485 - but no posted solution at least.

**Comment by bifa2** [comment]
I'll close this off as I manually edited the database instead.

**Comment by morpheus65535** [comment]
Did you try to simply force run the Sync with Radarr task? Sorry for the delay, work is crazy...

**Comment by bifa2** [comment]
> Did you try to simply force run the Sync with Radarr task? Sorry for the delay, work is crazy...

No worries! We all have a day job and have things to cram in before the holiday break. 

Yesterday I did try 'Update Movie list from Radarr' and also 'Update all Movie Subtitles from disk' but no joy. 

I just ran 'Update all Movie Subtitles from disk' once more (after unticking 'Use Embedded Subtitles' as I want it to recognise external subs) and it seemed like mass edit returned all films - certainly WAY more than previously... until I tried once again and it was back to the partial load. I tried another full update from disk just incase that helped but nope. 

Can mass edit be used for anything other than changing the profile? I think not in which case I won't need to use it again, but yeah, this is an odd bug. If you really want to spend time digging into it, happy to help further if I can, but equally happy for the case to remain closed! Thanks for all your work on this product btw :)
